apps/prairielearn/src/lib/editors.ts

```diff
@@ -1116,6 +1116,36 @@ export class QuestionDeleteEditor extends Editor {
   }
 }
 
+export class QuestionBatchDeleteEditor extends Editor {
+  private questions: Question[];
+
+  constructor(params: BaseEditorOptions & { questions: Question[] }) {
+    super(params);
+
+    this.questions = params.questions;
+    this.description = `Delete questions ${this.questions.map((x) => x.qid).join(', ')}`;
+  }
+
+  async write() {
```

Comment:
We could set up a QuestionDeleteEditor for each question and then call write() for each, but that seems inefficient and I don't think the QuestionDeleteEditor.write() method is going to change much.

Reply:
Another approach would be to update `QuestionDeleteEditor` to support either a single question or an array of questions. That should cut down on code duplication.

Reply:
Switched to that plan in fd0f052.